doc/development-context-aware-applications/how-to-update-and-query-context-information.md

```diff
@@ -66,101 +81,201 @@ it in the following way:
-          "value": "4.2"
-        },
+      "contextElement":
         {
-          "name": "location",
-          "type": "location",
-          "value": "40.419697, -3.691281",
-          "metadatas": [
-          {
-            "name": "location",
-            "type": "string",
-            "value": "WGS84"
-          }
-        },
-        {
-          "name": "postal_address",
-          "type": "string",
-          "value": "Calle Alcala 57, 28028 Madrid, Spain"
+          "type": "Restaurant",
+          "isPattern": "false",
+          "id": "Elizalde",
+          "attributes":
+          [
+            {
+              "name": "address",
+              "type": "",
+              "value":
+              {
+                "type": "postalAddress",
+                "streetAddress": "Cuesta de las Cabras Aldapa 2",
+                "addressRegion": "Araba",
+                "addressLocality": "Alegría-Dulantzi",
+                "postalCode": "01240"
+              }
+            },
+            {
+              "name": "aggregateRating",
+              "type": "",
+              "value":
+              {
+                "reviewCount": 1,
+                "ratingValue": 3
+              }
+            },
+            {
+              "name": "capacity",
+              "type": "PropertyValue",
+              "value": 120,
+              "metadatas":
+              [
+                {
+                  "name": "name",
+                  "type": "",
+                  "value": "capacity"
+                }
+              ]
+            },
+            {
+              "name": "department",
+              "type": "",
+              "value": "Franchise3"
+            },
+            {
+              "name": "description",
+              "type": "",
+              "value": "Restaurante de estilo sidrería ubicado en Alegria-Dulantzi. Además de su menú del día y carta, también ofrece menú de sidrería. El menú del día cuesta 9 euros. Los fines de semana la especialidad de la casa son las alubias con sacramentos. En lo que a bebidas se refiere, hay una amplia selección además de la sidra. Cabe destacar que se puede hacer txotx. La capacidad del establecimiento es de 50 personas pero la sidrería no dispone de aparcamiento.%5cn%5cnHORARIO: %5cn%5cnLunes a domingo: 9:00-17:00 y 19:00-23:00."
+            },
+            {
+              "name": "occupancyLevels",
+              "type": "PropertyValue",
+              "value": 0,
+              "metadatas":
+              [
+                {
+                  "name": "timestamp",
+                  "type": "",
+                  "value": ""
+                },
+                {
+                  "name": "name",
+                  "type": "",
+                  "value": "occupancyLevels"
+                }
+              ]
+            },
+            {
+              "name": "position",
+              "type": "coords",
+              "value": "42.8404625, -2.5123277",
+              "metadatas":
```

Comment:
The usage of `location` to specify entity location is going to be deprecated by Orion 1.1.0 (released by the end of April), see https://github.com/telefonicaid/fiware-orion/pull/2071. Thus, I'd recommend to use the "new way" in the examples (based in using `geo:point` as attribute type).